From aea9f8d156c3d68448959eecd094bcc6fa9f0221 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Fri, 27 Jul 2012 13:38:03 +0800
Subject: [PATCH 319/486] ENGR00215174 make fb0, fb2 and fb4 have
 independently display mode filter file.

make fb0, fb2 and fb4 have independently display mode filter file.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 mx6/libgralloc_wrapper/display_mode.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mx6/libgralloc_wrapper/display_mode.cpp b/mx6/libgralloc_wrapper/display_mode.cpp
index 2e9a7e8..b667611 100644
--- a/mx6/libgralloc_wrapper/display_mode.cpp
+++ b/mx6/libgralloc_wrapper/display_mode.cpp
@@ -176,9 +176,9 @@ static int read_graphics_fb_mode(int fb)
         //int size;
         memset(conf_modes, 0, sizeof(conf_modes));
         memset(&g_config_mode[0], 0, sizeof(g_config_mode));
-        int fd = open("/system/etc/display_mode.conf", O_RDONLY, 0);
+        int fd = open("/system/etc/display_mode_fb0.conf", O_RDONLY, 0);
         if(fd < 0) {
-            LOGE("Warning: /system/etc/display_mode.conf not defined");
+            LOGE("Warning: /system/etc/display_mode_fb0.conf not defined");
         }
         else {
             size = read(fd, conf_modes, sizeof(conf_modes));
-- 
1.8.0

